app/services/testdata/candidate/fsb/FsbResultEnteredStatusGenerator.scala

```diff
@@ -38,8 +40,20 @@ object FsbResultEnteredStatusGenerator extends FsbResultEnteredStatusGenerator {
 trait FsbResultEnteredStatusGenerator extends ConstructiveGenerator {
   val fsbTestGroupService: FsbService
 
-  def generate(generationId: Int, createCandidateData: CreateCandidateData)
+  def generate(generationId: Int, createCandidateDataIn: CreateCandidateData)
               (implicit hc: HeaderCarrier, rh: RequestHeader): Future[CreateCandidateResponse] = {
+
+    val createCandidateData = if (createCandidateDataIn.fsbTestGroupData.isEmpty) {
+      val schemeTypes = createCandidateDataIn.schemeTypes.getOrElse(List(SchemeId("HumanResources")))
```

Comment:
I'd suggest getting a random scheme from the DataFaker helper as it is driven from the schemes yaml file so this won't break if HR is removed for whatever reason.